src/ui/components/questions/questionsubmissioncomponent.js

```diff
@@ -168,6 +168,15 @@ const DEFAULT_CONFIG = {
     phrase: 'We\'re sorry, an error occurred.'
   }),
 
+  /**
+   * The default email error text, shown when there is an email is invalid with the QA Submission
+   * request.
+   * @type {string}
+   */
+  emailInvalidErrorText: TranslationFlagger.flag({
+    phrase: 'Your email address was not valid. Please refresh the page and try submitting your question again.'
```

Comment:
what do you think about changing that first sentence to: "Invalid email address"?

Reply:
that sounds fine too